setup.py

```diff
@@ -54,6 +54,7 @@
           "tqdm",
           "pathlib2; python_version<'3'",
           "backports.tempfile; python_version<'3.4'",
+          "ruamel.yaml"
```

Comment:
`ruamel.yaml` doesn't support Python 2.7 anymore since version [0.17.2](https://pypi.org/project/ruamel.yaml/0.17.2/). Is that version sufficient?

Reply:
I can't see why not at the moment. I'll test it tomorrow and see if something breaks